toot/tui/utils.py

```diff
@@ -161,3 +163,13 @@ def resize_image(basewidth: int, baseheight: int, img: Image) -> Image:
     if img.mode != 'P':
         img = img.convert('RGB')
     return img
+
+
+def can_render_pixels():
+    # temp image to make AutoImage happy
+    img = Image.new("RGB", (16, 16))
+    ai = AutoImage(img)
+
+    # BlockImage renders to half-cells
+    # KittyImage and ITerm2Image render to pixels
+    return not isinstance(ai, BlockImage)
```

Comment:
1. Instead of instantiating an image, I would suggest using [`auto_image_class()`](https://term-image.readthedocs.io/en/latest/reference/image.html#term_image.image.auto_image_class) (was previously named `auto_style()` but future plans necessitated the change) with `GraphicsImage` (instead of `BlockImage` just to simplify the logic):
```
return is_subclass(auto_image_class(), GraphicsImage)
```

GraphicsImage is the baseclass of all image classes implementing graphics-based render styles. See the [class heirachy](https://term-image.readthedocs.io/en/latest/reference/image.html#image-classes).

Reply:
Done.